riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/RiptideSettings.java

```diff
@@ -23,6 +23,11 @@
     private Defaults defaults = new Defaults();
     private GlobalOAuth oauth = new GlobalOAuth();
     private Map<String, Client> clients = new LinkedHashMap<>();
+    private Boolean mocked = false;
+
+    public void setMocked(final Boolean mocked) {
```

Comment:
You should get this for free.

Reply:
Top Level of the Settings currently only has setters generated.

Reply:
You mean getters, right? Isn't the constructor enough?

Reply:
I did not get it working without this `setter` somehow. He does not find a proper "writable" property path.